plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -243,12 +243,21 @@ public function register_ptk_patterns() {
 	 * @return array The parsed patterns.
 	 */
 	private function parse_categories( array $patterns ) {
-		if ( ! isset( $patterns['categories'] ) || ! is_array( $patterns['categories'] ) ) {
-			return array();
-		}
 
 		return array_map(
 			function ( $pattern ) {
```

Comment:
Again I'd type check `$pattern` is an array here before operating

Reply:
@tjcafferkey Done with 39e8cf8